resources/js/components/ui/form/SegmentedControl.module.scss

```diff
@@ -9,14 +9,30 @@
   [data-part="item"] {
     @apply relative select-none;
     @apply px-2 py-0.5 flex items-center;
-    @apply text-black cursor-pointer;
+    @apply text-black;
 
     &[data-state="unchecked"] {
-      @apply text-opacity-70 hover:text-opacity-100;
+      @apply text-opacity-70;
     }
 
     &[data-disabled] {
-      @apply text-opacity-50;
+      // @apply text-opacity-50;
```

Comment:
tu coś trzeba by postanowić, napisać czemu zakomentowane albo wyrzucić

Reply:
Dodałem, nie pamiętam czemu to było wykomentowane...